frontend/src/scenes/session-recordings/player/metaLogic.test.ts

```diff
@@ -64,7 +67,10 @@ describe('metaLogic', () => {
                 },
             })
             sessionRecordingsTableLogic().mount()
+
+            silenceKeaLoadersErrors()
```

Comment:
Note: Same here, the changes in this file also just clean up some test error logs that are un-related to this PR, but needed cleaning